packages/icons/src/Icon.module.css

```diff
@@ -0,0 +1,21 @@
+.hop-icon {
+    display: inline-block;
+    pointer-events: none;
+    height: var(--hop-space-240);
```

Comment:
Is it necessary given that each icon will receive a size?

Reply:
This isn't necessary, as the values are defined in the svg's `width` and `height`.